AK/CopyMethod.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2023, Ben Wiederhake <BenWiederhake.GitHub@gmx.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+// Because this interacts with ErrorOr and includes Error.h, this cannot go into TypedTransfer.h.
+
+#include <AK/Error.h>
+#include <AK/Format.h>
+#include <AK/Platform.h>
+
+namespace AK {
+
+// TODO:
+// - return decltype of clone() in FallibleCopyHelper
+// - Allow passing template arguments to FallibleCopyHelper
+// - Check that it works for clone-only types
```

Comment:
Are these TODOs still relevant? If so, please rewrite them into `FIXME`s. The last one sounds as if it should be part of this PR?